index.dd

```diff
@@ -94,10 +94,11 @@ $(DIVC boxes,
                 http://youtube.com/playlist?list=PL3jwVPmk_PRxo23yyoc0Ip_cP3-rCm7eB,
                 this YouTube playlist).
             )
-            $(P Stay updated with
-                $(LINK2 http://arsdnet.net/this-week-in-d,
-                    <cite>This Week in D</cite>) -
-                $(SPANC twid)
```

Comment:
Let's keep This Week in D. It's pretty cool, despite not having the "official" stamp on it. Just move it below the official blog. May be something like:

> Stay updated with the [official D Blog](https://dlang.org/blog/) and [This Week in D](http://arsdnet.net/this-week-in-d):
> * [D Blog - Compile-Time Sort in D](http://dlang.org/blog/2017/06/05/compile-time-sort-in-d/) by Michael Parker from June 5, 2017.
> * [This Week in D - May 28](http://arsdnet.net/this-week-in-d/2017-may-28.html): Blog article on optimization and Andrei Israel trip, Beta D 2.074, tip on mixin w/ anon func.